events/api.py

```diff
@@ -1416,6 +1416,23 @@ class LanguageViewSet(JSONAPIViewMixin, viewsets.ReadOnlyModelViewSet):
 register_view(LanguageViewSet, "language")
 
 
+class OrganizationFilter(django_filters.rest_framework.FilterSet):
+    dissolved = django_filters.BooleanFilter(
```

Comment:
I think the name is a bit ambiguous. Now `dissolved=true` returns all organizations, but `dissolved=false` filters them out. Is that the way the filter is supposed to work? A better name in that case would probably be `include_dissolved`, but then again that doesn't sound quite right, since by default they're already included.

Maybe reverse the logic and `hide_dissolved` would be a better filter?

Reply:
While `hide_dissolved` is of course perfectly fine, in cases like this my favourite would be `dissolved=true|false` where `true` means only the dissolved ones. That way all three cases (dissolved, not dissolved, both) would be covered with the single boolean filter. 

With just `hide_dissolved` it is not possible to get only the dissolved ones, and adding support for that later is a bit cumbersome with the `hide_dissolved` filter already being there. And being more general-purpose and easily extendable is a notch more important in these few APIs of ours that actually have also some outside users 😄

Reply:
Yes, was thinking about it to work like @tuomas777 said. I'd say I'll tweak the logic to include only dissolved ones when `true` and exclude dissolved when `false`.
Yes? 👍 👎